nautobot/virtualization/models.py

```diff
@@ -460,13 +460,15 @@ def clean(self):
             )
 
     def to_objectchange(self, action):
+
         # Annotate the parent VirtualMachine
         return ObjectChange(
             changed_object=self,
             object_repr=str(self),
             action=action,
-            related_object=self.virtual_machine,
             object_data=serialize_object(self),
+            object_datav2=serialize_object_v2(self),
+            related_object=self.virtual_machine,
         )
```

Comment:
```python
return super().to_objectchange(action, related_object=self.virtual_machine)
```

Reply:
the reason decided not to use `super(). to_objectchange()`in some of `to_objectchange()` method is because I get this error
```
{"error": "'super' object has no attribute 'to_objectchange'", "exception": "AttributeError", "nautobot_version": "1.3.0a0", "python_version": "3.6.15"}
```

Reply:
Hah - looks like `VMInterface`, `CircuitTermination`, `ComponentTemplateModel`, `ComponentModel`  aren't actually `ChangeLoggedModel` subclasses. I'd forgotten that `to_objectchange` is used sometimes in a duck-typing way independent of inheritance from `ChangeLoggedModel`.

I might argue that the correct fix here would be to make these classes actually inherit from `ChangeLoggedModel`, but I'm not certain why this is currently the way it is. @jathanism @lampwins any thoughts here?

Reply:
We discussed further and decided that changing the class inheritance is out of scope for this PR - I'll be opening up a separate housekeeping issue to address that tech debt. The current situation of having one-off implementations of `to_objectchange` is fine for now. Sorry for any trouble!

Reply:
#1424.